helper/cleartext_test.go

```diff
@@ -45,11 +45,5 @@ func TestSignClearText(t *testing.T) {
 	if err != nil {
 		t.Fatal("Cannot parse message:", err)
 	}
-	assert.Exactly(t, canonicalizeAndTrim(inputPlainText), string(clearTextMessage.GetBinary()))
-}
-
-func canonicalizeAndTrim(text string) string {
```

Comment:
I think the refactored code change is not the same as the original regex. 

In the original implementation, only the trailing space of the entire string input is trimmed, and then all the newlines are globally replaced with \r\n.

But, in the refactored implementation, spaces are trimmed on every line.

Reply:
This is intended behaviour according to RFC 4880 paragraphs 5.2.1 and 7.1